specification/network/resource-manager/Microsoft.Network/stable/2022-05-01/publicIpAddress.json

```diff
@@ -595,6 +638,28 @@
         }
       },
       "description": "Contains the IpTag associated with the object."
+    },
+    "PublicIpDdosProtectionStatusResult": {
+      "type": "object",
+      "properties": {
+        "publicIpAddressId": {
+          "type": "string",
+          "description": "Public IP ARM resource ID"
+        },
+        "publicIpAddress": {
+          "type": "string",
+          "description": "IP Address of the Public IP Resource"
+        },
+        "isWorkloadProtected": {
+          "type": "boolean",
+          "description": "Value indicating whether the IP address is DDoS workload protected or not."
```

Comment:
consider making this an enum : 

Explanation here : 
[https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html)

Replace boolean/switch properties with better enum
A Boolean will forever have two valid values (true or false). A string enum type is always preferred. Also, properties should always provide better values just than True and False. For example two switches "isTypeA" and "isTypeB" should be replaced with one enum "type": [A, B, DefaultType]. Even if you still believe [True, False] are the correct values for a property, you should use a string enum with values [True, False] instead of boolean. Enums are always a more flexible and future proof option because they allow additional values to be added in the future in a non-breaking way, e.g. [True, False, Unknown].